### PR TITLE
[FSD-1750] Add documentation for new honor_exclusivity flag and backfill some missing GET params

### PR DESCRIPTION
**Pull request by acdameli**
related to: https://github.com/stagebloc/core/pull/5151

### CODE REVIEW

File: api_v1.md [hunk 1]
```diff
@@ -716,6 +736,19 @@ This endpoint can used to list all the audio for a certain account.
...
+> <p> pass with value 1 to enable exclusive audio support. only audio
+> <p> which should be visible to the active user based on the audio's
+> <p> exclusivity setting and the user's fanclub status.
+> <p> requests without authentication or with non-fans will receive only
+> <p> non-exclusive audio.
+> <p> requests identifying a fanclub member on tier 1, 2, or 3 receive
+> <p> audio exclusive to their fanclub tier as well as non-exclusive
+> <p> audio.
+> <p> requests identifying an account administrator will receive all audio
+> <p> regardless of their exclusivity.
```
Comment (stagebloc-iceman) [inline api_v1.md]: How about...

```markdown
`honor_exclusivity`
> <p> whether to include content exclusive to a specific tier based on the active user's permissions
> <p> accepted values are 0 to only show public content and 1 to also include exclusive content.
> <p> defaults to 0
```

The description of expected results is a bit wordy and applies to all sections; perhaps we create a separate section to explain exclusivity and link there instead of doing copy-paste?

```markdown
> <p> requests without authentication or with non-fans will receive only non-exclusive content.
> <p> requests identifying a fanclub member will receive content exclusive to their fanclub tier as well as non-exclusive content.
> <p> requests identifying an account administrator will receive all content regardless of its exclusivity.
```

Thoughts? /cc @jdhartley @joshholat

Comment (joshholat) [inline api_v1.md]: Yeah, the general explanation of exclusivity could be put at either https://www.fullscreendirect.com/developers/api/v1/general-information or https://www.fullscreendirect.com/developers/api/v1/authentication. I think the shorted version here specifically is better.

Comment (acdameli) [inline api_v1.md]: unfortunately the general rule is not true across all content items. Without authentication, all items are restricted as described EXCEPT store items and audio. But I will identify those differences.

I'll update this doc and identify where the outliers are.

**Comment by acdameli** [comment]
@stagebloc-iceman @joshholat The text is almost exactly the same, with the two special cases, store item and audio having slightly different text. I also identified in the authentication section the two cases which are special.
